lib/zstd.h

```diff
@@ -1063,6 +1066,7 @@ ZSTDLIB_API size_t ZSTD_sizeof_DDict(const ZSTD_DDict* ddict);
 /* Advanced parameter bounds */
 #define ZSTD_TARGETCBLOCKSIZE_MIN   64
 #define ZSTD_TARGETCBLOCKSIZE_MAX   ZSTD_BLOCKSIZE_MAX
+#define ZSTD_SRCSIZEHINT_MAX        INT_MAX
```

Comment:
We should also add `ZSTD_SRCSIZEHINT_MIN = 0` here, since we take an int and don't want negative values.